hamocc/mo_carchm.F90

```diff
@@ -765,20 +770,24 @@ subroutine carchm_kequi(temp,saln,prb,Kh0,K1,K2,Kb,Kw,Ks1,Kf,Ksi,K1p,K2p,K3p,Ksp
 
     ! Kh0 = [CO2]/ fCO2 (fCO2 = fugacity of CO2 in air)
     ! Weiss (1974), note this does not include a correction for the effect of moist air at 
-    ! air-sea interface [mol/kg/atm]
+    ! air-sea interface [mol/kg/atm], temp=[-1,45], saln=[0,45]
     nKhwe74 = ad1+ad2/tk100+ad3*log(tk100)+s*(bd1+bd2*tk100+bd3*tk100**2)
     Kh0     = exp( nKhwe74 )
     ! K1 = [H][HCO3]/[H2CO3]   ; K2 = [H][CO3]/[HCO3]
-    ! Millero p.664 (1995) using Mehrbach et al. data on seawater scale
-    K1 = 10**( -1.0 * ( 3670.7 * invtk - 62.008 + 9.7944 * dlogtk - 0.0118 * s + 0.000116 * s2 ) )
-    K2 = 10**( -1.0 * ( 1394.7 * invtk + 4.777 - 0.0184 * s + 0.000118 * s2 ) )
+    ! Waters et al. (2014) on total pH scale, temp=[0,45], saln=[0,45]
```

Comment:
here and below - do the comments provide the valid ranges for the formulations? - or should it be what comes from BLOM?